### DISCUSSION TITLE
use shapefile in cormas models

### DISCUSSION THREAD

**Discussion by ElCep** [cormas/cormas#776] [Ideas]
Can you imagine loading vector data into CORMAS (shp or geoJSON). This implies looking at the world other than as a grid ?

**Comment by SergeStinckwich** [comment]
I see two possible solutions here: 1) support continuous space in CORMAS or 2) rasterize shapefiles when importing them in CORMAS.

**Comment by ElCep** [comment]
Obviously the first possibility is more attractive to me :sunglasses:

**Comment by hernanmd** [comment]
Etienne you may like to know I migrated the [Shapes support to Pharo 8](https://github.com/hernanmd/Shapes) yesterday, from the methods comments I read it is possible to layer several .shp files although there are no examples yet.

Can you tell me from where in the UI would you expect to load the .shp files in CORMAS?
Or the steps to get the Load .shp from the UI.

**Comment by ElCep** [comment]
It would be great to have an input from @pierbo but from my point of view, it would be great to have the same semantics as when you load a regular grid. What do you think?
